ulib/LowStar.Monotonic.Buffer.fsti

```diff
@@ -1103,6 +1167,10 @@ let modifies_liveness_insensitive_region_mreference_weak
 : Lemma
   (requires (modifies l2 h h' /\ region_liveness_insensitive_locs `loc_includes` l2 /\ HS.live_region h (HS.frameOf x)))
   (ensures (HS.live_region h' (HS.frameOf x)))
+  [SMTPatOr [
+    [SMTPat (modifies l2 h h'); SMTPat (HS.live_region h (HS.frameOf x))];
+    [SMTPat (modifies l2 h h'); SMTPat (HS.live_region h' (HS.frameOf x))];
```

Comment:
Why is this lemma and its pattern specific to `HS.frameOf x`? Doesn't it hold true for any region loc?

Reply:
Any thoughts on why this is specific to frameOf?

Reply:
It should, I will try.